### ISSUE TITLE
No relevant accessibility measures for people with intellectual and learning disabilities

### ISSUE THREAD

**Issue by joshueoconnor** [w3c/wcag21#520]
Dear Sir, dear Madam,

Inclusion Europe, the European Association of People with Intellectual Disabilities and their Families, is very concerned that the final draft of the WCAG 2.1 includes again almost no relevant accessibility measures for people with intellectual and learning disabilities. We are indeed surprised that the Working Group shows such a disregard for the accessibility needs of such a large group of people with disablities and call upon you to end these discriminatory practices against people with intellectual disabilities.

Furthermore, we draw your attention to the fact that intransparent and unaccessible working methods exclude any person with intellectual or learning disability from participating actively in the formulation of the WCAG. We will take the opportunities in the discussions of the European Accessibility Act with European policy-makers to raise this point.

In the following, we reiterate our main proposals that we had submitted to you before:

Insert new Criterion 2.4.13: Navigation to plain language summary
The link to a Plain Language Summary page of a site is provided in the upper part of the entry page of a site through a non-verbal icon (such as the Easy-to-Read Logo at www.easy-to-read.eu)

Amend Success Criterion 3.1.2 Language of Parts
Add this text:
If a site contains any pages or summaries in plain language, each plain language passage in the content can be programmatically determined except for proper names, technical terms, words of indeterminate language, and words or phrases that have become part of the vernacular of the immediately surrounding text.

Insert new Criterion 3.1.7: Plain Language Summary of site
When a website requires for most of its pages a reading ability more advanced than the primary education level, a summary page that does not require reading ability more advanced than the primary education level, is available as supplemental content. This Plain Language Summary page includes:

    Information about the purpose of the site
    Summary explanations of and links to the 5 most used pages on the site
    Contact information 

The summary page is clearly identified by a non-verbal icon.

Insert new Criterion 3.1.8 Plain Language Summary of main pages
When the 10 mostly used pages of a website require a reading ability more advanced than the primary education level, alternative content is provided that does not require reading ability more advanced than the primary education level. This content is clearly identified by a non-verbal icon.

Insert new term in Glossary
Plain language or easy-to-read language is a text that does not require reading ability more advanced than the primary education level. It supports access of people with intellectual or learning disabilities to relevant contents, but can also enhances accessibility for non-native speakers of a language. 

We hope that it will be possible to take account of these accessibility needs of people with intellectual disabilities.

Yours sincerely,

Geert Freyhoff

**Comment by awkawk** [comment]
Thank you for the comment.

You make four main points:

The WCAG development process excludes people with intellectual or learning disabilities
Request for

-  three new Success Criteria and a glossary item
- Navigation to plain language summary
- Plain language summary
- Plain language summary of main pages
- Request for modification of SC 3.1.2

WCAG 2.1 doesn’t include measures for people with intellectual and learning disabilities
It has been very important to the Working Group that people with various types of cognitive disabilities are included in the work of the group. We formed a task force for cognitive accessibility and welcomed many new members into the working group, many of whom either have some type of cognitive disability or work regularly on the topic. There is no question that the process of developing a specification with a group of almost 100 people providing input can be taxing due to the volume of email, wiki page updates, and spec updates that take place, but the W3C and the Working Group are committed to supporting the needs of any group members to help ensure that it is possible for anyone to participate to the greatest extent possible. Still, we recognize that there is always room for improvement and are open to any suggestions you may have. Although our COGA task force did a significant amount of background research, and an extensive Gap Analysis, it appears that a Success Criteria for a Plain Language Summary as found in your comment was not proposed by any stakeholders in the timeframe for WCAG 2.1. However, there is a proposed Success Criteria called Identify Common Purpose which may allow personalization and simplification of interfaces.

Regarding the inclusion of requirements for people with intellectual and learning disabilities, while we agree that the set of Success Criteria does not address every possible improvement that end users might benefit from, there are substantial challenges to ensure they meet the following acceptance criteria for Success Criteria both in WCAG 2.0 and 2.1:

- Be testable through automated or manual processes.
- Apply to all content (including many human languages - internationalization) unless preconditions for the application of the success criteria are explicitly identified.
- Apply across technologies to the greatest extent possible. (Technology-specific issues should usually be addressed in Techniques.)
- Have Success Techniques which demonstrate that each Success Criterion is implementable, using readily-available formats, user agents, and assistive technologies.

(See: https://www.w3.org/WAI/GL/wiki/WCAG_2.1_Success_Criteria.)

These are some of the core requirements for Success Criteria that are very important but do set a challenging bar for new criteria to meet. If Success Criteria are included that do not meet these requirements there is a strong risk that WCAG 2.1 might not gain consensus or might not be adopted. Of the currently 18 new Success Criteria, approximately half are viewed as providing some benefit for users with cognitive disabilities.

We should also mention that WCAG 2.0 provides important help for people with cognitive and learning disabilities. We hope that there will be significant advances in Assistive Technology for people with cognitive and learning disabilities in the future, which may facilitate further WCAG requirements upon authors and leverage more of the the existing requirements.

### Suggested new Success Criteria "Navigation to plain language summary"
The deadline for introducing new Success Criteria closed over the summer as a result of the tight Working Group Charter timeline. However, we will mark this issue with a label (“defer”) to ensure that we can review these at the right time when we are able to review new proposals. This also impacts the suggestion for a definition since that definition is related to the proposed new criteria.

We can provide it to the Cognitive Task Force (COGA) for consideration in a future version. It would impact the most important part of most web template designs and we expect there would be significant response from industry, so any objective research and studies your team can provide may help the COGA team with justifying this UI requirement universally across all web sites may be helpful.

### The suggested Amendment Success Criterion 3.1.2 Language of Parts
Amending existing WCAG 2.0 Success Criteria is not permitted for WCAG 2.1 development, so in addition to the timeline consideration above, this suggestion would not be possible during this round of development.

We can provide it to the Cognitive Task Force (COGA) for future consideration. In addition to the concerns about impact on site design, there is currently no assistive technology we know of that could make use of programmatically determined summaries in plain language. We suggest that this be demonstrated to the COGA team.

### Insert new Criterion 3.1.7: Plain Language Summary of site
We extensively examined several proposals for plain language requirements in WCAG 2.1. We went through many iterations and we were unable to find a way that they could meet the internationalization requirements for an Success Criteria that could apply to "all content", and also there was some concern about requirements for freedom of expression. Regarding applying a plain language to a summary, these concerns may or may not be able to be overcome.

We can provide it to the Cognitive Task Force (COGA) for future consideration.

### Ensure the summary page is clearly identified by a non-verbal icon.
In addition to the timeline considerations above, we have had trouble gaining consensus on specific icons in the past. There are different cultures and different environments which may or may not approve of a particular icon design. We know of no such internationally recognized icon. WCAG strives to apply across the entire web to all countries and all languages, and so we'd like to hear about how we can ensure that a requirement for an icon could meet this need.

### Insert new term in Glossary
We extensively examined several proposals for "plain language" requirements in WCAG 2.1. We went through many iterations and we were unable to find a way that they could meet the internationalization requirements for a Success Criteria. Also, there was some concern about requirements for freedom of expression.

There is a strong possibility that there are ideas that the Working Group has simply not considered yet. Fortunately, the Working Group is committed to updating the specification far more rapidly than has been done in the past (targeting every 2 years), so these ideas and others will be able to be considered more fully in the near future.
